compliance/cis_k8s/rules/cis_1_2_25/rule.rego

```diff
@@ -4,16 +4,16 @@ import data.compliance.cis_k8s
 import data.compliance.lib.common
 import data.compliance.lib.data_adapter
 
-# Ensure that the --audit-log-maxsize argument is set to 100 or as appropriate (Automated)
+# Ensure that the --request-timeout argument is set as appropriate (Automated)
 
 # evaluate
 process_args := data_adapter.process_args
 
 default rule_evaluation = false
 
 rule_evaluation {
```

Comment:
> But, setting this timeout limit to be too large can exhaust the API server resources making it
prone to Denial-of-Service attack.

I think that the rule needs to verify that the request timeout doesn't exceed 60 seconds.

Reply:
let's open a ticket for anything unrelated to the issue scope you find